apps/client/src/app/app.module.ts

```diff
@@ -25,7 +30,9 @@ import { CommonModule } from '@angular/common';
         HttpClientModule,
         CommonModule,
     ],
-    providers: [],
+    providers: [
+        { provide: LOCALE_ID, useValue: 'de' }
```

Comment:
There is an environment variable you can use from the file `.env`: "APP_LOCALE"

Reply:
Angular does not support accessing environment variables from .env or .env.local files directly like Node.js does. I could handle it in the environment.ts and environment.prod.ts, but then it is not in sync with the backend. I could also write a script to sync the values. Is that really necessary to have those two in sync?

Reply:
Doesn't need to, `process.env.APP_LOCALE` will be inserted by webpack (node.js) during compilation time. At least that's how it is usually working 🤔

Reply:
But we can do that later :)